### PR TITLE
Closes #362

### LINKED ISSUE #362: Models: calculation of prob and dose for a single model parameters
To do:

- [x] add a new vignette "Migrating from old `crmPack`" and include a section on this there
- [x] add to introduction vignette

With the new representation of models, it may be unclear to users how a probability can be derived for a certain dose and vice versa with fixed model parameters (a single parameter).
After discussion with @wwojciech, I was able to do the calculation, but I think that this should be put into the documentation (vignette?), so that end users are able to do the calculation. 

```
empty_data <- Data(doseGrid = c(1, 3, 5, 10, 15, 20, 25, 40, 50, 80, 100))

my_model <- LogisticNormal(
  mean = c(-0.85, 1),
  cov = matrix(c(1, -0.5, -0.5, 1), nrow = 2)
)

my_options <- McmcOptions(burnin = 10, step = 2, samples = 100)

samples <- mcmc(empty_data, my_model, my_options)

dose(0.3, my_model, samples)
my_dose <- doseFunction(my_model, alpha0 = mean(my_samples@data$alpha0), alpha1 = mean(my_samples@data$alpha1))
my_dose(0.3)

prob(10, my_model, samples)
my_prob <- probFunction(my_model, alpha0 = mean(my_samples@data$alpha0), alpha1 = mean(my_samples@data$alpha1))
my_prob(10)

```

### PR DESCRIPTION
**Pull request by wwojciech**
Closes #362

### CODE REVIEW

File: vignettes/dose-prob_funs.Rmd [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Example of usage of dose and prob functions"
```
Comment (danielinteractive) [inline vignettes/dose-prob_funs.Rmd:2]: can you please make this more general? I would say e.g. "Migrating from crmPack version < ..." (input current CRAN version)
and then the dose/prob is a section in here. thereby we can add other topics later and don't get too many vignettes.

**Review by danielinteractive — COMMENTED** [review]
thanks Wojciech for starting this, please see inside

**Review by danielinteractive — APPROVED** [review]

**Comment by wwojciech** [comment]
@danielinteractive - just updated `NEWS` file, hope it is ok.

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Version 1.0.9000.9087
+* Created a new vignette which describes how to use certain functions and features
+  of the `crmPack`, which changed after package's general cleanup. Added chapter
+  about `dose` and `prob` functions.
```
Comment (danielinteractive) [inline NEWS.md:4]:
```suggestion
* Created a new vignette which describes how to use certain functions and features
  of `crmPack` after the major refactoring.
```

**Pull request merged**